README.md

```diff
@@ -0,0 +1,23 @@
+# Urban Airship Accengage Cordova Plugin
+
+This plugin supports Cordova apps running on both iOS and Android.
+
+### Issues
+
+Please visit http://support.urbanairship.com/ for any issues integrating or using this plugin.
+
+### Requirements:
+ - cordova >= 9.0.0
+ - cordova-ios >= 5.0.1
+ - cococapods >= 1.7.3
+ - urbanairship-cordova >= 13.1.0
```

Comment:
there is a way to make urbanairship-cordova a dependency in the plugin.xml

Reply:
Ok, I will add the dependencies to the plugin.xml